README.md

```
@@ -69,131 +68,32 @@ docker run -d -p 9001:9001 -p 9030:9030 --name tor -v /data/tor:/data svengo/tor
-`RELAY_BANDWIDTH_BURST=value`
-
-Maximum bandwidth burst for the relay (e.g., 200 KBytes). This allows short bursts above the average rate, but still limits the maximum rate.
-
-(Default: *empty*)
+svengo/tor uses several environment variables to generate the ``torrc-defaults``-file, the variables are set to reasonable defaults (see below). You can edit ``/data/torrc`` to your needs after the first start.
+
+| Variable | Default | Description |
+|----------|---------|-------------|
+| `ORPORT` | `9001` | Advertise this port to listen for connections from Tor clients and servers. Format: `[address:]PORT [flags]`. This option is required to be a Tor server. |
+| `DIRPORT` | `9030` | If this option is nonzero, advertise the directory service on this port. Format: `[address:]PORT [flags]`. |
+| `EXITPOLICY` | `reject *:* # no exits allowed` | Set an exit policy for this server. Each policy is of the form `accept[6]|reject[6] ADDR[/MASK][:PORT]`. If `/MASK` is omitted, then this policy just applies to the host given. Instead of giving a host or network you can also use `*` to denote the universe (0.0.0.0/0 and ::/128), or `*4` to denote all IPv4 addresses, and `*6` to denote all IPv6 addresses. `PORT` can be a single port number, an interval of ports `FROM_PORT-TO_PORT`, or `*` . If PORT is omitted, that means `*`. |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)